radicle-cli/src/commands/checkout.rs

```diff
@@ -0,0 +1,136 @@
+use std::ffi::OsString;
+use std::path::PathBuf;
+
+use anyhow::anyhow;
+use anyhow::Context as _;
+
+use radicle::prelude::*;
+use radicle::storage::WriteStorage;
+
+use crate::project;
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+pub const HELP: Help = Help {
+    name: "checkout",
+    description: "Checkout a radicle project working copy",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad checkout <id> [<option>...]
+
+Options
+
+    --no-confirm    Don't ask for confirmation during checkout
+    --help          Print help
+"#,
+};
+
+pub struct Options {
+    pub id: Id,
+}
+
+impl Args for Options {
+    fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)> {
+        use lexopt::prelude::*;
+        use std::str::FromStr;
+
+        let mut parser = lexopt::Parser::from_args(args);
+        let mut id = None;
+
+        while let Some(arg) = parser.next()? {
+            match arg {
+                Long("no-confirm") => {
+                    // Ignored for now.
```

Comment:
```suggestion
                    // Ignored for now.
                    unimplemented!()
```
Maybe throw this in there in case anyone tries to run it?